junit-jupiter-api/src/main/java/org/junit/jupiter/api/MethodOrderer.java

```diff
@@ -253,7 +253,7 @@ class Random implements MethodOrderer {
 
 		static {
 			DEFAULT_SEED = System.nanoTime();
-			logger.info(() -> "MethodOrderer.Random default seed: " + DEFAULT_SEED);
+			logger.config(() -> "MethodOrderer.Random default seed: " + DEFAULT_SEED);
```

Comment:
Hmm... I wonder if this is really a good idea. :thinking: What if the user (=us) want to actually have the random seed logged? It can be incredibly useful when trying to reproduce a particular failure where tests (incorrectly) intererfer with each other. In cases like that, you typically want to re-run the tests with the exact same random seed.

Would `logger.debug` be an option here, or is there any other way to get these `logger.config` statements to be propagated to e.g. Logback? /cc @sormuras @marcphilipp

Reply:
Sorry for the noise. This can still be configured by the user; they just have to enable this in their JUL config. If the `SLF4JBridgeHandler` is being used, `config` statements are converted to `debug` level because of implementation details in SLF4J. More details: https://jira.qos.ch/browse/LOGBACK-1612?focusedCommentId=21253&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21253